test/unit/RoundsManager.js

```diff
@@ -254,19 +253,22 @@ contract("RoundsManager", accounts => {
     describe("blockHash", () => {
         it("should fail if block is in the future", async () => {
             const latestBlock = await web3.eth.getBlockNumber()
-            await expectThrow(roundsManager.blockHash(latestBlock + 1))
+            // Note that current block = latestBlock + 1, so latestBlock + 2 is in the future
+            await expectRevertWithReason(roundsManager.blockHash(latestBlock + 2), "can only retrieve past block hashes")
         })
 
         it("should fail if the current block >= 256 and the block is more than 256 blocks in the past", async () => {
             await fixture.rpc.wait(256)
 
             const latestBlock = await web3.eth.getBlockNumber()
-            await expectThrow(roundsManager.blockHash(latestBlock - 257))
+            // Note that current block = latestBlock + 1, so latestBlock - 256 = current block - 257
+            await expectRevertWithReason(roundsManager.blockHash(latestBlock - 257), "can only retrieve hashes for last 256 blocks")
         })
 
         it("should fail if block is the current block", async () => {
             const latestBlock = await web3.eth.getBlockNumber()
-            await expectThrow(roundsManager.blockHash(latestBlock))
+            // Note that current block = latestBlock + 1
```

Comment:
Based on my previous comment, `latestBlock` would actually be the current block here.